mdflynn/rancid-tomatillos
# Feature/design movie card

## What is the change?
This adds styling for the individual movie page

## What does it fix?
This fixes the testing issues after adding React Router

## Is this a fix or a feature?
This is both a fix and a feature pr.

## Where should the reviewer start?
- Review the entire Movie file and the accompanying Scss file.
- Entire App.test.js file.

## How should this be tested?
- Clone down the repo
- Run `npm test` in the console to see test suite passing
- Run `npm start` and navigate to localhost:3000 to see current site design and feature.
